src/_gettsim/social_insurance_contributions/ges_pflegev.py

```diff
@@ -207,73 +208,34 @@ def ges_pflegev_beitr_rente_m(
     out = (
         _ges_krankenv_bemessungsgrundlage_rente_m
         * 2
-        * soz_vers_beitr_params["beitr_satz"]["ges_pflegev"]["standard"]
+        * sozialv_beitr_params["beitr_satz"]["ges_pflegev"]["standard"]
     )
 
     # Add additional contribution for childless individuals
     if ges_pflegev_zusatz_kinderlos:
         out += (
             _ges_krankenv_bemessungsgrundlage_rente_m
-            * soz_vers_beitr_params["beitr_satz"]["ges_pflegev"]["zusatz_kinderlos"]
+            * sozialv_beitr_params["beitr_satz"]["ges_pflegev"]["zusatz_kinderlos"]
         )
 
     return out
 
 
-def _ges_pflegev_beitr_midijob_sum_arbeitn_arbeitg_m_bis_09_2022(
-    midijob_bemessungsentgelt_m: float,
-    soz_vers_beitr_params: dict,
-    ges_pflegev_zusatz_kinderlos: bool,
-) -> float:
-    """Calculating the sum of employee and employer care insurance contribution for
-    midijobs until September 2022.
-
-    Parameters
-    ----------
-    midijob_bemessungsentgelt_m
-        See :func:`midijob_bemessungsentgelt_m`.
-    ges_pflegev_zusatz_kinderlos
-        See :func:`ges_pflegev_zusatz_kinderlos`.
-    soz_vers_beitr_params
-        See params documentation :ref:`soz_vers_beitr_params <soz_vers_beitr_params>`.
-
-    Returns
-    -------
-
-    """
-
-    gesamtbeitrag_midijob_m = (
-        midijob_bemessungsentgelt_m
-        * 2
-        * soz_vers_beitr_params["beitr_satz"]["ges_pflegev"]["standard"]
-    )
-
-    # Add additional contribution for childless individuals
-    if ges_pflegev_zusatz_kinderlos:
-        gesamtbeitrag_midijob_m += (
-            midijob_bemessungsentgelt_m
-            * soz_vers_beitr_params["beitr_satz"]["ges_pflegev"]["zusatz_kinderlos"]
-        )
-
-    return gesamtbeitrag_midijob_m
-
-
-def _ges_pflegev_beitr_midijob_sum_arbeitn_arbeitg_m_ab_10_2022(
+def _ges_pflegev_beitr_midijob_sum_arbeitn_arbeitg_m(
```

Comment:
Strange that we had two identical functions before..

Reply:
Mechanical application of the reform is the most likely explanation. The components all did change.